test/import.bats

```diff
@@ -541,6 +541,35 @@ newimports:
   run: |
     [ -f /stacker/imports/file1.txt ]
 EOF
+  stacker build
+}
 
+@test "importing container images" {
+    cat > stacker.yaml <<EOF
+cimg-import:
+    from:
+      type: oci
+      url: $CENTOS_OCI
+    import:
+    - path: docker://alpine:edge
+      dest: /
+    run: |
+      [ -d /var/lib/apk ]
+
+compose-img:
+    from:
+      type: oci
+      url: $CENTOS_OCI
+    import:
+    - path: docker://ghcr.io/homebrew/core/openssl/1.1:1.1.1k
+      dest: /
+    - path: docker://ghcr.io/homebrew/core/curl:8.0.1
+      dest: /opt/
```

Comment:
how does this work?
You take a layer that is rooted at "/" and place it at "/opt" ? Won't that have to create a new layer (sha256 blob tarball) , thus reducing the benefit and destroying any signatures on the imported layer?